_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -64,9 +66,9 @@ Both of the above two sections have been focused on increasing flow of business
 Is this a competency required for the duration of the project? If not, a new team member is not the optimal choice, as it is preferable to not interchange members of the team frequently.
```

Comment:
wow there are three 'nots' in that second sentence! ;-) Are we saying that if increasing flow is not required then fewer changes to the personnel on the team is best, or are we saying the opposite, that if increasing flow is required then fewer changes to the personnel on the team is best.

Reply:
There are not too many nots in there! ;) 

Increasing flow is always important! 

I shall try to clarify ...